homeassistant/components/intellifire/strings.json

```diff
@@ -5,23 +5,36 @@
       "manual_device_entry": {
         "description": "Local Configuration",
         "data": {
-          "host": "[%key:common::config_flow::data::host%]"
+          "host": "[%key:common::config_flow::data::host%] (IP Address)"
+        }
+      },
+      "api_config": {
+        "title": "IntelliFire - API Configuration",
+        "description": "IntelliFire will need to reach out to [iftapi.net](https://iftapi.net/webaccess/login.html) in order to obtain an API key. Once it has obtained this API key, the rest of its interactions will occur completely within the local network. If the API key were to expire it would again need to reach out to https://iftapi.net/webaccess/login.html\n\nUsername and Password are the same information used in your IntelliFire Android/iOS application. ",
```

Comment:
This description is providing implementation details that are not necessary. I would scrap it.
```suggestion
```

Reply:
OkieDokie!